packages/auth-kit/jest.setup.ts

```diff
@@ -0,0 +1,3 @@
+import { TextEncoder, TextDecoder } from 'util'
+
+Object.assign(global, { TextDecoder, TextEncoder })
```

Comment:
A comment on why this is needed would be helpful

Reply:
Got this error before:
```
 FAIL  src/packs/monerium/MoneriumPack.test.ts
  ● Test suite failed to run

    ReferenceError: TextEncoder is not defined

    > 1 | import { Hash, encodeFunctionData, hashMessage } from 'viem'
```

TextEncoder is used by viem but not bundled with it. Therefore we need to polyfill it for the tests.

Also added this as a comment to the code.